.bithoundrc

```diff
@@ -9,11 +9,14 @@
      ],
      "ignore": [
         "src/TestFrameworkOrchestrator.ts",
+        "src/SandboxCoordinator.ts",
+        "src/MutantTestMatcher.ts",
```

Comment:
Should we make an issue so we eventually remove these from the ignore list once the new syntax is supported?